src/services/sync/strategies/CSVSyncer.js

```diff
@@ -394,6 +397,36 @@ class CSVSyncer extends mix(Syncer).with(ClassImporter) {
 	static splitClasses(classes) {
 		return classes.split('+').filter((name) => name !== '');
 	}
+
+	static assertDateFormat(dateInput) {
+		const bdayErrors = [];
+		if (!dateInput) {
+			// todo: change to logger
+			bdayErrors.push('missing birthday value');
+			return bdayErrors
+		}
+		if (typeof dateInput !== 'string') {
+			bdayErrors.push('incorrect values, birthday must be a string');
+			return bdayErrors
+		}
+		// https://stackoverflow.com/questions/15491894/regex-to-validate-date-format-dd-mm-yyyy
+		const dateValidationRegex = /^(?:(?:31(\/|-|\.)(?:0?[13578]|1[02]))\1|(?:(?:29|30)(\/|-|\.)(?:0?[13-9]|1[0-2])\2))(?:(?:1[6-9]|[2-9]\d)?\d{2})$|^(?:29(\/|-|\.)0?2\3(?:(?:(?:1[6-9]|[2-9]\d)?(?:0[48]|[2468][048]|[13579][26])|(?:(?:16|[2468][048]|[3579][26])00))))$|^(?:0?[1-9]|1\d|2[0-8])(\/|-|\.)(?:(?:0?[1-9])|(?:1[0-2]))\4(?:(?:1[6-9]|[2-9]\d)?\d{2})$/;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 362. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5220139)